### PR TITLE
1261 mirth local

### PR DESCRIPTION
**Pull request by Orta21**
Ref: #1261

### Description

IHE Local Setup OSS

### Release Plan

- [ ] Merge once approved

### CODE REVIEW

File: packages/ihe-server/README.md
Comment (leite08) [inline packages/ihe-server/README.md:1]: I think we're missing some steps here, no? Doesn't one has to perform some operations after starting the docker container?

File: packages/ihe-server/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+version: "3"
+services:
+  ihe-server:
+    image: nextgenhealthcare/connect
```
Comment (Goncharo) [inline packages/ihe-server/docker-compose.yml:4]: needs to be `nextgenhealthcare/connect:4.2.2` to match with the plugins

File: packages/ihe-server/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  ihe-postgres:
+    image: postgres:14.4-alpine
+    container_name: ihe-postgres
+    restart: always
+    environment:
+      POSTGRES_DB: "db"
+      POSTGRES_USER: "admin"
+      POSTGRES_PASSWORD: "admin"
+    ports:
+      - "5435:5432"
```
Comment (Goncharo) [inline packages/ihe-server/docker-compose.yml:35]: let's use `"5436:5432"`, we already have 5435 taken up by another service

**Review by Goncharo — CHANGES_REQUESTED** [review]

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**